### PR TITLE
[v2.10] RKE2 September patch

### PR DESCRIPTION
**Pull request by rafaelbreno**

### CODE REVIEW

File: channels-rke2.yaml [hunk 1]
```diff
@@ -2164,6 +2164,38 @@ releases:
         repo: rancher-rke2-charts
         version: 0.1.1800
     featureVersions: *featureVersions-v1
+  - version: v1.28.14+rke2r1
+    minChannelServerVersion: v2.8.3-alpha1
+    maxChannelServerVersion: v2.9.99
```
Comment (ShylajaDevadiga) [inline channels-rke2.yaml]: maxChannelServerVersion: v2.10.99 for v2.10

File: channels-rke2.yaml [hunk 2]
```diff
@@ -2244,6 +2276,38 @@ releases:
         repo: rancher-rke2-charts
         version: 0.1.1800
     featureVersions: *featureVersions-v1
+  - version: v1.29.9+rke2r1
+    minChannelServerVersion: v2.9.0-alpha1
+    maxChannelServerVersion: v2.9.99
```
Comment (ShylajaDevadiga) [inline channels-rke2.yaml]: Same maxChannelServerVersion: v2.10.99

File: channels-rke2.yaml [hunk 3]
```diff
@@ -2332,6 +2396,38 @@ releases:
         repo: rancher-rke2-charts
         version: 0.1.1800
     featureVersions: *featureVersions-v1
+  - version: v1.30.5+rke2r1
+    minChannelServerVersion: v2.9.0-alpha1
+    maxChannelServerVersion: v2.9.99
```
Comment (ShylajaDevadiga) [inline channels-rke2.yaml]: maxChannelServerVersion: v2.10.99

File: channels-rke2.yaml [hunk 4]
```diff
@@ -2382,3 +2478,32 @@ releases:
         repo: rancher-rke2-charts
         version: 0.1.1800
     featureVersions: *featureVersions-v1
+  - version: v1.31.1+rke2r1
+    minChannelServerVersion: v2.9.0-alpha1
+    maxChannelServerVersion: v2.9.99
```
Comment (ShylajaDevadiga) [inline channels-rke2.yaml]: maxChannelServerVersion: v2.10.99

File: channels-rke2.yaml (hunk 1 above)
Comment (rafaelbreno) [reply channels-rke2.yaml]: Done!

**Review by ShylajaDevadiga — APPROVED** [review]

**Pull request merged**